api/src/main/java/net/starype/quiz/api/game/QuizGame.java

```diff
@@ -0,0 +1,17 @@
+package net.starype.quiz.api.game;
+
+
+import net.starype.quiz.api.game.player.UUIDHolder;
+
+public interface QuizGame {
+
+    void start();
+    void pause();
+    void resume();
+    void forceStop();
+
+    boolean isCurrentRoundFinished();
+    void nextRound();
+
+    void sendInput(UUIDHolder player, String message);

```

Comment:
### To do
- [ ] Adding print answer support (or interface to communicate such information to the discord implementation)